importlib_resources/readers.py

```diff
@@ -72,6 +72,10 @@ def __init__(self, *paths):
         if not all(path.is_dir() for path in self._paths):
             raise NotADirectoryError('MultiplexedPath only supports directories')
 
+    @property
+    def paths(self):
+        return self._paths.copy()
```

Comment:
If we type-annotate this property to be a non-mutable Sequence, and because `self._paths` is not mutated internally (also treated like a non-mutable Sequence), there's no need to make a copy. But I'm okay either way.